internal/config/5_integration_defaults.go

```diff
@@ -0,0 +1,10 @@
+//go:build integration && !test
+// +build integration,!test
+
+package config
+
+import "github.com/RHEnVision/provisioning-backend/internal/config/parser"
+
+func init() {
+	parser.Viper.Set("featureFlags.environment", "test")
+}
```

Comment:
I did a lot of cleanup in these files, I renamed them to `_defaults` because they really only contain default values. Also added integration Go build tag which is same as test tag at the moment.